float32[] x
float32[] mu
float32[] obs
---
# result
float32[] y_out
float32[] var
bool result
